doc/v2/documentation_data_format.rst

```diff
@@ -14,187 +15,375 @@ This document explains the PEtab data format.
+  - ``NON_PARAMETER_TABLE_ID``: A valid PEtab ID referring to a constant or
+    differential entity (:ref:`v2_model_entities`), including PEtab output
+    parameters, but excluding parameters listed in the
+    :ref:`v2_parameters_table` (independent of their ``estimate`` value).
 
 
 Changes from PEtab 1.0.0
 ------------------------
-
 PEtab 2.0.0 is a major update of the PEtab format. The main changes are:
 
-* Support for non-SBML models
-* Clarification and specification of various previously underspecified aspects
-  (math expressions, overriding values in the condition table, etc.)
-* Support for extensions
-* Observable IDs are now allowed to be used in observable/noise formulas
+* Support for models in other formats than SBML (:ref:`v2_model`).
+* Revised condition table format from wide to long format
+  (:ref:`v2_conditions_table`).
+* ``simulationConditionId`` and ``preequilibrationConditionId`` in the
+  :ref:`v2_measurements_table` are replaced by ``experimentId`` and a more
+  flexible way for defining experiments and time courses
+  (:ref:`v2_experiments_table`).
+* Support for math expressions in the condition table
+  (:ref:`v2_conditions_table`).
+* Clarification and specification of various previously underspecified
+  aspects, including overriding values via the condition table
+  (:ref:`v2_math_expressions`).
+* Support for format extensions.
+* Observable IDs can now be used in observable and noise formulas
+  (:ref:`v2_observables_table`).
+
+.. _v2_model:
+.. _v2_model_entities:
 
 Model definition
 ----------------
 
-PEtab 2.0.0 is agnostic of specific model formats. A model file is referenced
-in the PEtab problem description (YAML) via its file name or a URL.
+PEtab 2.0.0 is **model format agnostic**, meaning it does not depend on a
+specific model description. The model file is referenced in the :ref:`PEtab
+problem description (YAML) <v2_problem_yaml>` by its file name or a URL.
 
-Condition table
----------------
+PEtab distinguishes between three types of entities:
+
+* **Differential entities**: Entities that are defined in terms of a
+  time-derivative, e.g., the targets of SBML rate rules or species that change
+  due to participation in reactions (reactants or products).
 
-The condition table specifies parameters, or initial values of species and
-compartments for specific simulation conditions (generally corresponding to
-different experimental conditions).
+* **Algebraic entities**: Entities that are defined in terms of an algebraic
+  assignment rather than time derivatives. They are not necessarily constant,
+  for example, the targets of SBML assignment rules.
 
-This is specified as a tab-separated value file in the following way:
+* **Constant entities**: Entities are that are defined in terms of a constant
+  value but may be subject to event assignments, e.g., parameters of an SBML
+  model that are not targets of rate rules or assignment rules.
```

Comment:
It's unclear that algebraic assignments like SBML initialAssignmentRules are constant entities. I'm not sure how to best word that here. In PEtab SciML there is an attempt at distinguishing these by calling this "prior to simulation". I guess the difference between algebraic entities and constant entities is that constant entities are piecewise constant (e.g. initialAssignmentRules or eventAssignments can affect them, but otherwise they are constant), and algebraic entities are everything else.

Reply:
The thing that is unclear to me is that both of these are algebraic, but one is compatible with "constant entity" whereas the other is not:
- initialAssignmentRule is an algebraic equation that is evaluated at one time point
- assignmentRule is an algebraic equation that is evaluated throughout time

Reply:
I tried to rephrase it, please check again.

Reply:
Looks good, the intention of the different entities is clear now :+1: Thanks